src/main/java/com/indico/IndicoClient.kt

```diff
@@ -42,6 +42,7 @@ interface IndicoClient: AutoCloseable {
     /**
      * Execute a mutation to extract information from a document.
      */
+    @Deprecated("Not supported")
```

Comment:
This is enough for deprecating, but we should put something descriptive for the consumers that directs them to an alternative functionality. Maybe something like `This feature will be removed in a future version. Use Workflows instead`.

Reply:
Ah yep good call out. I updated with description to redirect them to workflow submission.